### ISSUE TITLE
Support `nodejs16.x` function runtime

### ISSUE THREAD

**Issue by domeq** [DataDog/serverless-plugin-datadog#265]
## Expected Behavior
Since AWS, Serverless and non-serverless `dd-trace` support Node 16 runtime, it should be possible to use it with `serverless-plugin-datadog` too.

I'm trying to do it with latest version of the plugin `v5.0.1` and failing.

## Actual Behavior
When trying to deploy a serverless application with config like this:

```
service: foobar
plugins:
  - serverless-plugin-datadog

provider:
  name: aws
  runtime: nodejs16.x
  region: eu-central-1
  endpointType: regional

custom:
  datadog:
    service: foobar
    env: development
    version: 1.0.0
```

I'm getting the following error:
```
Auto instrumenting functions with Datadog
Adding Lambda Library Layers to functions
Unable to add Lambda Layers to function *** with runtime nodejs16.x
```

Application deploys and is working, but I have no monitoring anymore.

## Steps to Reproduce the Problem

  1. Try deploying a serverless application with `nodejs16.x` runtime
  2. Watch output for the `Unable to add Lambda Layers to function *** with runtime nodejs16.x` errors
  3. Verify that all monitoring is now gone.

## Specifications

  - Serverless Framework version: `3.17.0`
  - Datadog Serverless Plugin version: `5.0.1`
  - Lambda function runtime (Python 3.7, Node 10, etc.): `nodejs16.x`

**Comment by astuyve** [comment]
Hi @domeq - thanks for reaching out. This was just merged and should be released soon! https://github.com/DataDog/datadog-lambda-js/pull/285

**Comment by astuyve** [comment]
This is released in [5.1](https://github.com/DataDog/serverless-plugin-datadog/releases/tag/v5.1.0)
